Antidote/MainWindowController.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  MainWindowController.swift
+//  Antidote
+//
+//  Created by Yoba on 17/12/2016.
+//  Copyright © 2016 Dmytro Vorobiov. All rights reserved.
+//
+
+import Cocoa
+
+/**
+    MainWindowController is an entry point in application. At least until "tutorialWindowController" is created.
+ 
+    It contains NSSplitViewController. At left part there is always contact list.
+ 
+    At the right part will be shown one of the controllers: InitialViewController at launch, when no contacts selected; MessagesViewController when contacts is selected; CallViewController when user wishes to call someone.
+ */
+
+// MARK: init methods and properties
+
+class MainWindowController: NSWindowController {
+    struct Constants {
```

Comment:
Probably fileprivate (applies to `Constants` in all files, except for `BaseConstants`).